README.md

```diff
@@ -26,6 +26,49 @@ make generate build
 ./bin/ocis -h
 ```
 
+## Getting Started
```

Comment:
I'm not sure if this would match better for the docs folder.

Reply:
hmm... agree, maybe this way people will actually read the docs!